common/types/post.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+// PostConfig is the configuration of the Post protocol, used for data creation, proofs generation and validation.
+type PostConfig struct {
```

Comment:
i don't understand this move. 

common/types, as it stand today, are for data. options specific to packages reside in separate package.
this move makes the code organization more confusing. what about other options scattered in its own package?

please undo this move. and maybe use a separate PR to bring up discussion for this purpose.

Reply:
This is a necessity to do what @dshulyak asked me to do: move the mocks to a `mocks` package.

If the PostConfig stays in the `activation` package I cannot create a mock using that type since then `mocks` would depend on the `activation` package and `activation` package would depend on `mocks` causing a cyclic dependency.

Reply:
Obsolete, mocks have been moved in https://github.com/spacemeshos/go-spacemesh/pull/3598